src/components/ProductListPage/components/AllItemList.tsx

```diff
@@ -16,23 +17,20 @@ const getPageSize = () => {
     return 10;
   }
 };
+
 function AllItemList() {
-  const [itemList, setItemList] = useState([]);
-  const [page, setPage] = useState(1); // 현재 페이지
-  const [pageSize, setPageSize] = useState(getPageSize());
-  const [totalPages, setTotalPages] = useState(0);
-  const [order, setOrder] = useState("recent");
+  const [itemList, setItemList] = useState<Item[]>([]);
+  const [page, setPage] = useState<number>(1); // 현재 페이지
+  const [pageSize, setPageSize] = useState<number>(getPageSize());
+  const [totalPages, setTotalPages] = useState<number>(0);
+  const [order, setOrder] = useState<"recent" | "favorite">("recent");
 
-  const handleSort = (e) => {
-    const userSelect = e.target.value;
-    if (userSelect === "recent") {
-      setOrder("recent");
-    } else {
-      setOrder("favorite");
-    }
+  const handleSort = (e: ChangeEvent<HTMLSelectElement>) => {
+    const userSelect = e.target.value as "recent" | "favorite";
```

Comment:
직접 작성하기보다는 아래와 같이 타입을 만들어서 사용하면 좋을 것 같습니다. 

```suggestion
    const userSelect = e.target.value as SelectType;
```